lib/plugins/aws/provider.js

```diff
@@ -1584,6 +1585,11 @@ class AwsProvider {
     }).then((result) => result.StackResourceDetail.PhysicalResourceId);
   }
 
+  isDeploymentBucketVersioningEnabled() {
```

Comment:
Let's remove that util.

We can unconditionally check for `this.serverless.service.provider.deploymentBucketObject.versioning`